service_mesh/service_mesh_install/installing-ossm.adoc

```diff
@@ -4,7 +4,7 @@ include::modules/ossm-document-attributes.adoc[]
 :context: installing-ossm
 toc::[]
 
-Installing the {ProductShortName} involves installing the Operator, and then creating and managing a custom resource definition file to deploy the control plane.
+Installing the {ProductShortName} involves installing the Operator, creating and managing a custom resource definition file to deploy the control plane, and creating a `ServiceMeshMemberRoll` to specify the namespaces associated with the {ProductShortName}.
```

Comment:
Do we want to change "the Operator" to specify that (for production) we're actually installing four Operators?